system/hardware/tici/agnos.json

```diff
@@ -17,7 +21,11 @@
     "size": 274432,
     "sparse": false,
     "full_check": true,
-    "has_ab": true
+    "has_ab": true,
+    "alt": {
```

Comment:
why do these partitions also have alts?

Reply:
Only because they are gzips. I'm happy to remove and continue changing the file extension like before if you want

Reply:
oh, think it's best to just do the replace with that. at some point, think we want to fix the JS xz extract and don't want to do a manifest update when we ship that

Reply:
I'll remove the .gz from the manifest, including changing the alt system image to .xz, and flash can handle it